src/pages/NewBeer.js

```diff
@@ -0,0 +1,77 @@
+import { Button, Col, Divider, Input, Row } from 'antd'
+import React from 'react'
+import { useNavigate } from 'react-router-dom'
+import { createNewBeer } from '../service/services'
+
+function NewBeer() {
+
+  const navigate = useNavigate()
+
+  async function createBeer(e) {
+    e.preventDefault()
+
+    const name = e.target[0].value
+    const tagline = e.target[1].value
+    const description = e.target[2].value
+    const first_brewed = e.target[3].value
+    const brewers_tips = e.target[4].value
+    const attenuation_level = e.target[5].value
+    const contributed_by = e.target[6].value
+
```

Comment:
what if the form is about 50 inputs ? this is not dinamic has to have handle change